pkgs/development/tools/symfony-cli/default.nix

```diff
@@ -12,6 +12,12 @@ buildGoModule rec {
     sha256 = "sha256-xYNq5hnkk6OIsnDHhN/vh1LNuP7isTNgtTY2WUwC8x4=";
   };
 
+  postInstall = ''
+    runHook preInstall
```

Comment:
Those hooks are not required.

Reply:
Ok, removing them now.

Reply:
Done.

Reply:
Out of curiosity, why they are no required?

Reply:
They are only required in installPhase and similar.